src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -662,7 +662,7 @@ public void DigestValue_CRLF()
             
             Assert.Equal("IKbfdK2/DMfXyezCf5QggVCXfk8=", Convert.ToBase64String(digest));
 
-            X509Certificate2 cert = new X509Certificate2(_pkcs12, "PLACEHOLDER");
+            X509Certificate2 cert = new X509Certificate2(_pkcs12, "mono");
```

Comment:
While you are editing this - For this and other changes, should we just add the suppression comment now ? All other crypto/hash testing files have an exception approved already // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Not a secret.")]

Reply:
Best would be cherrypick https://github.com/dotnet/corefx/pull/43055 once we have all exemptions (probably we already have)